Add docs about kingfisher process integration (#406)
open-contracting/kingfisher-collect: docs/local.rst

--------

You should now have a crawl directory within the ``data`` directory containing OCDS files. For help using data, read about `using open contracting data <https://www.open-contracting.org/data/data-use/>`__.

Integrate with `Kingfisher Process <https://kingfisher-process.readthedocs.io/>`_

> --------

Reviewer: Have the dashes align with the heading text. We can remove the link from the heading, since there is a link in the first paragraph.

Besides storing the scraped data on disk, you can also send them to an instance of `Kingfisher Process <https://kingfisher-process.readthedocs.io/>`_ for processing.

> To do that, you need to have deployed an instance of `Kingfisher Process web app <https://kingfisher-process.readthedocs.io/en/latest/web.html#web-app>`_ and set the following environment variables:

Reviewer: ```suggestion
To do that, you need to deploy an instance of Kingfisher Process, including its `web app <https://kingfisher-process.readthedocs.io/en/latest/web.html#web-app>`__. Then, set the following either as environment variables or as Scrapy settings in ``kingfisher_scrapy.settings.py``:
```

> * ``KINGFISHER_API_URI``: The url where Kingfisher Process `web app <https://kingfisher-process.readthedocs.io/en/latest/web.html#web-app>`_ is served
> * ``KINGFISHER_API_KEY``: The Api Key configured for Kingfisher Process `web app <https://kingfisher-process.readthedocs.io/en/latest/config.html#web-api>`_

Reviewer: ```suggestion
``KINGFISHER_API_URI``
  The URL from which Kingfisher Process' `web app <https://kingfisher-process.readthedocs.io/en/latest/web.html#web-app>`_ is served
``KINGFISHER_API_KEY``
  The `API key <https://kingfisher-process.readthedocs.io/en/latest/config.html#web-api>`__ that was configured for Kingfisher Process' web app
```

If we link "web app" both times, users will assume the link is the same. I also used a definition list instead.

> For example:

Reviewer: ```suggestion
For example, set the environment variables, then run ``scrapy crawl`` commands:
```

.. code-block:: bash

>     export KINGFISHER_API_URI = 'http://127.0.0.1:5000'
>     export KINGFISHER_API_KEY = 1234

Reviewer: ```suggestion
   export KINGFISHER_API_URI='http://127.0.0.1:5000'
   export KINGFISHER_API_KEY=1234
   scrapy crawl my_spider
```
